lib/cli.rb

```diff
@@ -1,19 +1,25 @@
-class Cli
+require_relative './content.rb'
+require_relative './game.rb'
+
+class Cli 
   def call
     clear_screen
-    puts menu
-    puts select_marker
-  end
-
-  def menu
-    "Welcome to Tic Tac Toe!\n"
+    display_text(Content.welcome)
+    display_text(Content.select_marker)
+    user_input 
   end
 
-  def select_marker
-    "Choose 'X' or 'O' to play game: \n"
-  end
+  def display_text(text)
+    puts text + "\n\n"
+  end 
 
   def clear_screen
     system("clear")
   end
+
+  def user_input 
+    gets.chomp
+    Game.new.empty_board 
```

Comment:
so you get the input....then make an empty board? 🤔 Seems a bit odd

Reply:
The idea there was to get the user to choose their marker ‘X’ or ‘O’, and then the empty board is displayed, next step would be to play a turn. any thoughts what I could do different there? An idea would be to first display the empty board and then ask the user whether they would like to choose X or O - choosing X can allow the user to have the first turn and start the game.

Reply:
@ssunday I moved user_input from Cli to Game, Game now has a call method that will call user_input and then display_board (empty board to start with).  commit 419cb4933d9da60a8fcd82b535700caf5fd638f5